doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,27 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-1 v2-6-0
+
+KRKNWK-18965: Malformed group messages can cause memory leak
+  Matter accessories utilizing group communication may experience memory leaks if the secure group message is malformed.
+
+  **Workaround:** Manually cherry-pick commit to ``sdk-connectedhomeip`` from the upstream `Matter SDK PR #32970 <https://github.com/project-chip/connectedhomeip/commit/bdf3e6e183cba3d13bc5103bf014b47841a14de1>`
+
+.. rst-class:: v2-6-1 v2-6-0
```

Comment:
```suggestion
.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
```
Since it's the same overall issue, I think it's better to have it as one entry.